tools/linter_lib/custom_check.py

```diff
@@ -842,17 +844,20 @@
         },
         {
             "pattern": "su zulip -c [^']",
+            "exclude": {"api_docs/"},
             "include_only": {"docs/"},
```

Comment:
This has an `include_only`, so I don't think you need the exclude. Same with the next two.

Reply:
Looking at the documentation comment for `include_only` in [zulint custom_rules.py](https://github.com/zulip/zulint/blob/80aba3fd88faf6598665ac670a1d30f3c028df0d/zulint/custom_rules.py#L165):
```python
        # 'include_only': 'set([<path>, ...])' - includes only those files where <path> is a
        #                                        substring of the filepath.
```

So, `"docs/"` is a substring of `"api_docs/"` hence the custom rules here needed to be updated.

This was caught when I opened the PR and CI was failing on the backend lint step. I put up the current change quickly as it was the end of my day, so it wasn't well thought through as a solution. Will look more at it today.